package.json

```diff
@@ -100,14 +100,16 @@
     "compile": "webpack --mode development --watch",
     "build": "webpack --mode development",
     "build:enterprise": "webpack --mode development --env enterprise",
+    "chat:build": "cd chat && yarn && yarn build",
     "test:copyassets": "ncp ./src/test/fixture ./out/test/fixture/",
-    "prettier": "prettier --write src/",
-    "prettier:check": "prettier --check src/",
+    "prettier": "prettier --write src/ chat/",
+    "prettier:check": "prettier --check src/ chat/",
```

Comment:
mistake?

Reply:
why mistake?

Reply:
this is the main `package.json`.. you're removing prettier from all other `src/`?

Reply:
no, just added the /chat to it.
![image](https://github.com/codota/tabnine-vscode/assets/10290661/87e17960-99ab-4777-9ac9-d7d47ccb78df)

Reply:
Oh sorry. Missed the space